scalajvm/test/ApproverModelSpec.scala

```diff
@@ -1,23 +0,0 @@
-import org.specs2.mutable._
```

Comment:
This is not run in any step of CI. This is hard to support, because it requires separate database. This is not even a unit test. We are not gonna need this.

Actually, there is not so much to cover with unit tests in our project, because we don't have that much logic. We just mapping database to html. What are we testing? Database? The Play framework?

Currently we need only some basic integration tests. There is an issue for this #28 and I'm planning to work on it soon.